Extension/src/LanguageServer/client.ts

```diff
@@ -995,6 +998,7 @@ export class DefaultClient implements Client {
         const settings_dimInactiveRegions: boolean[] = [];
         const settings_enhancedColorization: string[] = [];
         const settings_suggestSnippets: (boolean | undefined)[] = [];
+        const settings_caseSensitiveFileSupport: (string | undefined)[] = [];
```

Comment:
I don't think this variable is used any more.

Reply:
you're right its removed